recipe/meta.yaml

```diff
@@ -9,13 +8,14 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: e1431d84a9cfb23e6773e72078ce8e65cad6745816d4cbf9ae67da5ea419acda
 
 build:
   ignore_run_exports:
     - lz4-c  # [win]
   number: 0
-  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation --ignore-installed .
+  skip: True  # [py<37]
 
 requirements:
   build:
```

Comment:
host should be:
```
- python
- pip
- setuptools
- setuptools_scm
- toml
- wheel
- pkgconfig
- lz4-c 1.9.4
```

https://github.com/python-lz4/python-lz4/blob/v4.3.2/pyproject.toml#L2

pytest runner is not required because we don't ask it to run the tests at build time.
https://github.com/python-lz4/python-lz4/blob/v4.3.2/setup.py#L164-L166

Reply:
Okay, thanks! I knew I shouldn't have second-guessed myself because I did not think it was required at build time either.